### PR TITLE
Display a notification when map instantiation fails

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
Part of https://github.com/CartoDB/cartodb/issues/7985. This implements notifications for "map instantiation error".

Basically, when there's an unknown error, the following notification is displayed:

screen shot 2016-07-13 at 18 08 15

Right now, the notification is not closable, but I can change this. The notification will be automatically closed when the next request to the Maps API succeeds. (cc: @saleiva, makes sense? how do you like the copy?)

@nobuti can you CR please? Thx!

### CODE REVIEW

File: lib/assets/locale/en.json [hunk 1]
```diff
@@ -1474,6 +1474,9 @@
     }
   },
   "notifications": {
+    "vis": {
+      "failed": "This map couldn't be rendered. Please try to refresh the page and contact us if the error persists."
```
Comment (nobuti) [inline lib/assets/locale/en.json]: Not sure if we need some styles here, @piensaenpixel?

File: lib/assets/javascripts/cartodb3/vis-notifications.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    var error = vis.get('error');
+    if (error) {
+      this._showVisErrorNotification();
+    } else {
+      this._hideVisErrorNotification();
+    }
+  },
+
+  _showVisErrorNotification: function () {
+    Notifier.addNotification(this._getNotificationAttrs());
```
Comment (nobuti) [inline lib/assets/javascripts/cartodb3/vis-notifications.js]: Should we need to check if the notification is already open?

**Comment by nobuti** [comment]
Just 2 doubts, the rest is 💯

File: lib/assets/javascripts/cartodb3/vis-notifications.js (hunk 2 above)
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb3/vis-notifications.js]: Notification has an ID so the Notifier won't create more than one of these notifications 😄

**Comment by piensaenpixel** [comment]
I think we should add "This map couldn't be rendered" in bold and red.

File: lib/assets/javascripts/cartodb3/vis-notifications.js [hunk 3]
```diff
@@ -32,7 +34,10 @@ var VisNotifications = {
     return {
       id: NOTIFICATION_ID,
       status: 'error',
-      info: _t('notifications.vis.failed'),
+      info: NOTIFICATION_TEMPLATE({
```
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb3/vis-notifications.js]: cc: @xavijam

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/vis-notifications.js]: 👍

**Comment by alonsogarciapablo** [comment]
screen shot 2016-07-14 at 12 26 34

**Pull request merged**